src/eduid_webapp/email/validators.py

```diff
@@ -32,12 +32,12 @@
 #
 
 from marshmallow import ValidationError
-from flask import request
-from eduid_common.api.utils import get_dashboard_user
+from flask import request, current_app
+from eduid_common.api.utils import get_user
 
 
 def validate_email(email):
-    user = get_dashboard_user()
+    user = get_user(current_app.email_proofing_userdb)
```

Comment:
Shouldn't user here be the authorized user returned from the central db?

I don't think you ever need to read the user from the proofing_userdb, only write it and sync to the central db.